SPECS-CROSS/binutils_cross/binutils_cross.spec

```diff
@@ -2,8 +2,8 @@
 # These should be set programatically in the future.
 %global _host_arch      x86_64
 %global _target_arch    aarch64
-%global _tuple          %{_target_arch}-linux-gnu
-%global _cross_name     %{_target_arch}-linux-gnu
+%global _tuple          %{_target_arch}-%{_vendor}-linux-gnu
```

Comment:
What does `%{_vendor}` currently defined as in our environment?

Reply:
It resolves to "mariner", as defined by `--with-vendor="mariner"` during our rpm package build.

Note: `%{_host_vendor}` however is incorrect (x86_64-pc-, or aarch64-unknown-) since it is something pulled from autoconf.

Reply:
Sounds good! Thanks for the explanation!